### ISSUE TITLE
Hippocratic license not considered free by Debian, GNU

### ISSUE THREAD

**Issue by Apteryks** [vcr/vcr#959]
Hi!

Have you considered that since switching to the Hippocratic license `vcr` won't be able to be distributed (packaged) by Debian and other GNU/Linux distributions that comply with free software distribution guidlines (GNU FSDG).  Particularly, the Hippocratic license considered non-free because it "restricts what jobs people can use the software for", per https://www.gnu.org/licenses/license-list.html#hippocratic.

This means that test suite of software included in Debian and FSDG-compliant distributions will need to be disabled or switch to something else than `vcr`. There's this Debian bug about it: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=984689).

Thanks for your time!

**Comment by krainboltgreene** [comment]
Yes, it was considered, and per the link they pinned to a version prior to the change.

I don't think they should be distributing it at all though, it makes no sense.

**Comment by Apteryks** [comment]
Hi, and thanks for the answer.

So to clarify my understanding, allow me to paraphrase what you wrote:

1. The incompatibility with the free software licenses was already known when choosing to switch to the Hippocratic license, but that wasn't considered a blocker.
2. GNU/Linux distributions following the GNU FSDG should stop distributing VCR

Did I get this right?

If so, feel free to close the issue.  It's unfortunate, but the choice is yours.

**Comment by krainboltgreene** [comment]
I mean you’re welcome to make the case that we should follow Debian’s desire to be able to use software for the effort of violating human rights, the case that the license restricts, but man I wouldn’t be so loud about it. It looks terrible in the news headlines.

**Comment by Apteryks** [comment]
The problem with creating new incompatible licenses is that there's no end in sight: I could fork the Hippocratic license and add some new incompatible clause, such as "this software must not be used for activities leading to global warming" and (apart from being very difficult to quantify/determine), would lead to yet more fragmentation of the free software world.  

Your mind seems made up already, so I won't argue on the license choice,  but I wanted to point out that not being in favor of using the Hippocratic license does not mean I'm in favor of violating human rights.  There's more to it than this.

**Comment by andrewfader** [comment]
Have you considered that the hippocratic clause is probably legally unenforceable? And you're making it harder to use vcr in the supply chain of open source projects that require their licenses to be permissive and non-restrictive? E.g. https://github.com/coreinfrastructure/best-practices-badge/issues/1712

**Comment by aalirezatajik33** [comment]
> Hi!
> 
> Have you considered that since switching to the Hippocratic license `vcr` won't be able to be distributed (packaged) by Debian and other GNU/Linux distributions that comply with free software distribution guidlines (GNU FSDG). Particularly, the Hippocratic license considered non-free because it "restricts what jobs people can use the software for", per https://www.gnu.org/licenses/license-list.html#hippocratic.
> 
> This means that test suite of software included in Debian and FSDG-compliant distributions will need to be disabled or switch to something else than `vcr`. There's this Debian bug about it: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=984689).
> 
> Thanks for your time!
